servers/services/src/main/proto/table.proto

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Is this file new? I am wondering if this was moved from somewhere else, there should be a delete/add diff?

Reply:
Very good point. It is a file with protobuf entities definitions. It was not removed from the older package. Will fix.